cibuild.sh

```diff
@@ -324,9 +342,8 @@ function usage {
 
 function enable_ccache {
   export USE_CCACHE=1;
-  export CCACHE_DIR=$prebuilt/ccache/.ccache;
   ccache -c
-  ccache -M 5G;
+  ccache -M 200M;
```

Comment:
Reduce this to keep to a size that we can likely cache all the parallel builds under the github limits.